File: SpringWeb/SpringWebTest-1/src/main/java/com/example/demo/main/hrd/entity/EmpBase.java
package com.example.demo.main.hrd.entity;


import org.hibernate.annotations.DynamicInsert;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "EMP_BASE")
@DynamicInsert
@Data
public class EmpBase {
	
	@Id
	@Column(name = "EMP_ID")
	private Long empId;

	@Column(name = "PASSWORD")
	private String password;

	@Column(name = "COMPANY_CD")
	private String companyCd;
	
	@Column(name = "EMP_NO")
	private Long empNo;

	@Column(name = "EMP_NAME")
	private String empName;
	
	@Column(name = "CTZ_NO", nullable = true)
	private String ctzNo;

	@Column(name = "EMAIL", nullable = true)
	private String email;

	@Column(name = "EMP_KIND_CD")
	private String empKindCd;

	@Column(name = "HIRE_CD")
	private String hireCd;
	
	@Column(name = "IN_OFF_YN")
	private String inOffYn;
	
	@Column(name = "GENDER_CD")
	private String genderCd;

	@Column(name = "BIRTH_YMD")
	private java.sql.Date birthYmd;
	
	@Column(name = "POS_CD", nullable = true)
	private String posCd;
	
	@Column(name = "ORG_CD", nullable = true)
	private String orgCd;
	
	@Column(name = "DUTY_CD", nullable = true)
	private String dutyCd;
	
	@Column(name = "JOB_CD", nullable = true)
	private String jobCd;
	
	@Column(name = "PAY_ORG_CD", nullable = true)
	private String payOrgCd;

	@Column(name ="TEMP_YN")
	private String tempYn;
	
	@Column(name = "HIRE_YMD", nullable = true)
	private java.sql.Date hireYmd;

	@Column(name = "RETIRE_YMD", nullable = true)
	private java.sql.Date retireYmd;

	@Column(name = "RETIRE_TYPE_CD", nullable = true)
	private String retireTypeCd;

	@Column(name = "CAREER_NUM", nullable = true)
	private Integer careerNum;

	@Column(name = "FILE_PATH_ID", nullable = true)
	private String filePathId;

	@Column(name = "MOD_USER_ID")
	private Long modUserId;

	@Column(name = "MOD_DATE")
	private java.sql.Date modDate;
	
	@Column(name = "TZ_CD")
	private String tzCd;

	@Column(name = "TZ_DATE")
	private java.sql.Date tzDate;
	

}

File: Spring_OAuth2/securityserver/src/main/kotlin/com/example/securityserver/model/domain/user/EmpBase.kt
package com.example.securityserver.model.domain.user;

import java.io.Serializable
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import java.sql.Date
@Entity
@Table(name = "EMP_BASE")
class EmpBase : Serializable{
	@Id
	var empId: Long? = null

	var userPassword: String? = null

	var companyCd: String? = null

	var empNo: Long? = null

	var ctzNo: String? = null

	var email: String? = null

	var empKindCd: String? = null

	var hireCd: String? = null

	var inOffYn: String? = null

	var genderCd: String? = null

	var birthYmd: Date? = null

	var posCd: String? = null

	var orgCd: String? = null

	var jobCd: String? = null

	var payOrgCd: String? = null

	var tempYn: String? = null

	var hireYmd: Date? = null

	var retireYmd: Date? = null

	var retireTypeCd: String? = null

	var careerNum: Int? = null

	var filePathId: String? = null

	var modUserId: Long? = null

	var modDate: Date? = null

	var tzCd: String? = null

	var tzDate: Date? = null
	override fun equals(other: Any?): Boolean {
		if (this === other) return true
		if (javaClass != other?.javaClass) return false

		other as EmpBase

		return empId == other.empId
	}

	override fun hashCode(): Int {
		return empId?.hashCode() ?: 0
	}

	override fun toString(): String {
		return "EmpBase(empId=$empId, userPassword=$userPassword, companyCd=$companyCd, empNo=$empNo, ctzNo=$ctzNo, " +
				"email=$email, empKindCd=$empKindCd, hireCd=$hireCd, inOffYn=$inOffYn, genderCd=$genderCd, " +
				"birthYmd=$birthYmd, posCd=$posCd, orgCd=$orgCd, jobCd=$jobCd, payOrgCd=$payOrgCd, " +
				"tempYn=$tempYn, hireYmd=$hireYmd, retireYmd=$retireYmd, retireTypeCd=$retireTypeCd, " +
				"careerNum=$careerNum, filePathId=$filePathId, modUserId=$modUserId, modDate=$modDate, " +
				"tzCd=$tzCd, tzDate=$tzDate)"
	}
}
